tests/unit_tests/serialization.cpp

```diff
@@ -671,12 +672,12 @@ TEST(Serialization, portability_wallet)
   const bool testnet = true;
   const bool restricted = false;
   tools::wallet2 w(testnet, restricted);
-  string wallet_file = epee::string_tools::get_current_module_folder() + "/../../../../tests/data/wallet_9svHk1";
+  const boost::filesystem::path wallet_file = unit_test::data_dir / "wallet_9svHk1";
   string password = "test";
   bool r = false;
   try
   {
-    w.load(wallet_file, password);
+    w.load(wallet_file.native(), password);
```

Comment:
Can you explain this please ? It looks like it ought to be a separate patch maybe,

Reply:
It's due to switching from string to boost::filesystem::path

Reply:
We could keep wallet_file as a string and convert up above where wallet_file is declared, but the current way seems marginally better because boost::filesystem::path is the natural type for wallet_file not string. Splitting hairs.

Reply:
OK, thanks.